make-emojis

```diff
@@ -37,6 +30,37 @@ custom_aliases.each do |custom, original|
     end
 end
 
+# @param file_name [String] a PNG filename, ex: '1f4aa-1f3ff.png'
+# @return [String] the PNG bytes
+def download_emoji_png(file_name)
+    source_url = "https://github.com/EmojiTwo/emojitwo/raw/master/png/#{file_name.downcase}"
```

Comment:
I'm pulling new emoji from EmojiTwo which is a clone of the last set of free emoji from EmojiOne. So we should be good from a license point of view.